plugins/parsers/binary/parser.go

```diff
@@ -16,7 +16,7 @@ import (
 
 type Parser struct {
 	AllowNoMatch bool            `toml:"allow_no_match"`
-	Endianess    string          `toml:"endianess" deprecated:"1.27.4;use 'endianness' instead"`
+	Endianness    string          `toml:"endianness" deprecated:"1.27.4;use 'endianness' instead"`
```

Comment:
No! Actually this is to _not_ break configs (i.e. what you should do for the other breaking config changes above), please leave this as-is!